kibana/Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV AOP_KIBANA_PLUGIN_VER=4.5.1 \
     CONFIG_PATH=/etc/kibana \
     NODE_ENV=production \
     KIBANA_VER=4.6.4 \
-    NODE_BIN=/usr/bin/node \
+    NODE_BIN=/opt/rh/rh-nodejs6/root/usr/bin/node \
```

Comment:
It is possible that following issue is only my local, as I was informed that mileage may vary when building rhel based images on Fedora, but is it possible that we may need to setup more ENV variables? 

Looking at what [scl enable](https://gist.githubusercontent.com/wozniakjan/321faa0f3163ef95f341867237c64369/raw/8064abfc7fea705b9719b0d6161033ce3c1aa3df/enable) does in this image, it may be insufficient to only change the path to nodejs binary. An [error message](https://gist.githubusercontent.com/wozniakjan/321faa0f3163ef95f341867237c64369/raw/8064abfc7fea705b9719b0d6161033ce3c1aa3df/log-install%2520failure) I see when building the image locally is a weak support but still may be worth addressing, both seem to relate to shared library path.

And it looks like kibana is still pulling own nodejs as a dependency in my case - https://gist.githubusercontent.com/wozniakjan/321faa0f3163ef95f341867237c64369/raw/8064abfc7fea705b9719b0d6161033ce3c1aa3df/log-dependencies